### PR TITLE
samples ipc_service: Convert to sysbuild

### PR DESCRIPTION
**Pull request by aescolar**
Convert these sample applications builds to sysbuild.

### CODE REVIEW

**Review by carlocaione — DISMISSED** [review]

File: samples/subsys/ipc/ipc_service/icmsg_me/CMakeLists.txt [hunk 1]
```diff
@@ -6,27 +6,14 @@
...
 
-set(REMOTE_ZEPHYR_DIR ${CMAKE_CURRENT_BINARY_DIR}/ipc_service_remote-prefix/src/ipc_service_remote-build/zephyr)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
 
-if("${BOARD}" STREQUAL "nrf5340dk_nrf5340_cpuapp")
-  set(BOARD_REMOTE "nrf5340dk_nrf5340_cpunet")
-else()
+if(NOT ("${BOARD}" STREQUAL "nrf5340dk_nrf5340_cpuapp"))
   message(FATAL_ERROR "${BOARD} is not supported for this sample")
 endif()
```
Comment (nordicjm) [inline samples/subsys/ipc/ipc_service/icmsg_me/CMakeLists.txt:14]: Would it make sense to make this only buildable using sysbuild?

File: samples/subsys/ipc/ipc_service/icmsg_me/sysbuild.cmake [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+ExternalZephyrProject_Add(
+	APPLICATION remote
+	SOURCE_DIR  ${APP_DIR}/remote
+	BOARD       ${SB_CONFIG_NET_CORE_BOARD}
+)
```
Comment (nordicjm) [inline samples/subsys/ipc/ipc_service/icmsg_me/sysbuild.cmake:8]: If that is the case then you can instead put an error here if `SB_CONFIG_NET_CORE_BOARD` is not set and means that only a mapping board list needs added to the Kconfig.sysbuild file

File: samples/subsys/ipc/ipc_service/icmsg_me/CMakeLists.txt (hunk 1 above)
Comment (aescolar) [reply samples/subsys/ipc/ipc_service/icmsg_me/CMakeLists.txt:14]: I'm honestly not sure. I mean, it is still possible to build each core image without sysbuild, and I don't know if we win anything by not supporting it. I would lean to keep supporting it.

**Comment by aescolar** [comment]
Now the 3 ipc_service samples have been ported to sysbuild :)

**Review by carlocaione — APPROVED** [review]

**Review by nordicjm — APPROVED** [review]

**Review by jhedberg — APPROVED** [review]

**Pull request merged**